src/gui/tray/Notification.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+
+// Custom qml modules are in /theme (and included by resources.qrc)
+import Style 1.0
+
+Item {
+    id: notificationItem
+    property alias text: notificationMessage.text
+    signal dismissNotification()
+
+    width: parent.width - Style.trayWindowBorderWidth*2
```

Comment:
Could you please clarify this? As I see it, I'm relying here on an external style property already (trayWindowBorderWidth) and only care about making it full-width of the tray window minus it's decoration border (whatever size it may be now or in the future). Guess I'm just don't getting it.

Reply:
I am with @nicolasfella here.
This sizing property should be set in the parent but not in the child. That makes it much easier to reuse and to maintain in the future. So move it to places where you use the Notification component.